legal-api/src/legal_api/models/business.py

```diff
@@ -86,6 +85,12 @@ def identifier(self, value: str):
         else:
             raise BusinessException('invalid-identifier-format', 406)
 
+    @property
+    def nextAnniversary(self):
+        arFilings = Filing.get_filings_by_type(self.id, Filing.FilingType.AR.value)
+        count = len(arFilings)
+        return self.founding_date+relativedelta(years=count+1)
+
```

Comment:
I had done that initially, but I realised last_ar_date doesn't necessarily track the anniversary (correct me if I'm wrong), but that is a good point I'll review the test scenarios with Nittin

Reply:
I'm also referring 2 things beyond that.
1. how things are counted. for a corp with 50years of history, all 48+ ARs are retrieved, serialized, sent across the network, deserialized, objects graphs created and then just size of the array is used? Either count on the server, or use founding date, the calculated field last_ar_date, etc.
2. we are currently using datedelta, not relativedelta. Always open to switching libraries, but since these 2 work a little differently it'll confuse future maintainers to have 2 packages to do the same, although slightly different things.